arroyo-api/queries/api_queries.sql

```diff
@@ -84,11 +84,21 @@ WHERE organization_id = :organization_id AND id = :id;
 
 ----------- pipelines -------------------
 
+--: DbPipelineRest (textual_repr?, udfs?)
+
+--! get_pipelines : DbPipelineRest
+SELECT pipelines.pub_id as id, name, type, textual_repr, udfs, program FROM pipelines
+WHERE pipelines.organization_id = :organization_id AND pipelines.pub_id IS NOT NULL;
```

Comment:
@mwylde since the REST API is based on the new public ids, I've excluded older pipelines that don't have one. Is that ok?

Reply:
I think that's fine